changelog.d/20241015_111820_max.tuecke_sc_35029_deprecate_delete_flag.md

```diff
@@ -0,0 +1,5 @@
+Added
+~~~~~~~
+
+- Added new `--delete-destination-extra` flag to `globus timer create transfer` and `globus transfer` that mirrors the existing `--delete` flags behaviors. (:pr:`1037`)
```

Comment:
Double backticks are used in restructured text to denote source code. Stephen's [changelog fragment](https://raw.githubusercontent.com/globus/globus-sdk-python/refs/heads/main/changelog.d/20240917_121333_sirosen_add_token_response_classes.rst) is an example.
```suggestion
- Added new ``--delete-destination-extra`` flag to ``globus timer create transfer`` and ``globus transfer`` that mirrors the existing ``--delete`` flags behaviors. (:pr:`1037`)
```